File: token-validation-spring/src/test/java/no/nav/security/token/support/spring/integrationtest/ProtectedApplication.java
package no.nav.security.token.support.spring.integrationtest;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ProtectedApplication {
    public static void main(String[] args) {
        new SpringApplication(ProtectedApplication.class).run(args);
    }
}

File: token-validation-spring/src/test/kotlin/no/nav/security/token/support/spring/integrationtest/ProtectedApplication.kt
package no.nav.security.token.support.spring.integrationtest

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class ProtectedApplication {
    fun main(args: Array<String>) {
        runApplication<ProtectedApplication>(*args)
    }
}